code/stable/gamephysics/SRS/HTML/GamePhysics_SRS.html

```diff
@@ -620,7 +620,7 @@ <h1>Introduction</h1>
           <div class="subsection">
             <h2>Purpose of Document</h2>
             <p class="paragraph">
-              The primary purpose of this document is to record the requirements of the game physics library. Goals, assumptions, theoretical models, definitions, and other model derivation information are specified, allowing the reader to fully understand and verify the purpose and scientific basis of Game Physics. With the exception of <a href=#Sec:SysConstraints>system constraints</a>, this SRS will remain abstract, describing what problem is being solved, but not how to solve it.
+              The primary purpose of this document is to record the requirements of Game Physics Library. Goals, assumptions, theoretical models, definitions, and other model derivation information are specified, allowing the reader to fully understand and verify the purpose and scientific basis of Game Physics. With the exception of <a href=#Sec:SysConstraints>system constraints</a>, this SRS will remain abstract, describing what problem is being solved, but not how to solve it.
```

Comment:
Hmm... I think here it makes sense to say "**the** Game Physics Library" - I wonder if we could "move" the decision of whether or not to include "the" to the examples' Body files, or include an option to include the word "program" (e.g., "the Projectile program"). (Regardless, this change should be pulled out to a separate PR to reduce the number of things that need discussion that are tackled in a single PR.)

Reply:
I agree with @samm82.